src/mpyl/steps/deploy/kubernetes.py

```diff
@@ -53,6 +55,16 @@ def execute(self, step_input: Input) -> Output:
                 endpoint = '/' if has_specific_routes_configured else '/swagger/index.html'
                 spec[DEPLOYED_SERVICE_KEY] = f'{hostname}{endpoint}'
             artifact = input_to_artifact(ArtifactType.DEPLOYED_HELM_APP, step_input, spec=spec)
+            if properties.target == Target.PRODUCTION:
```

Comment:
@koeves how do we test this?

Reply:
I will try to find a project that I can safely mess with in production (we have a mock service for example). Then I can check if the same image is deployed to test